lms/djangoapps/teams/tests/test_views.py

```diff
@@ -502,15 +502,25 @@ def test_bad_course_data(self, status, data):
         self.post_create_team(status, data)
 
     def test_missing_name(self):
-        self.post_create_team(400, {
-            'course_id': str(self.test_course_1.id),
-            'description': "foobar"
-        })
+        self.post_create_team(
```

Comment:
I don't understand why "missing name" corresponds to already being in a team in the course.